src/ansys/fluent/core/filereader/case_file.py

```diff
@@ -68,7 +68,12 @@ def numeric_value(self):
         float
             Numeric value of the Fluent input parameter.
         """
-        return float(self._component(0))
+        try:
```

Comment:
@prmukherj this doesn't look like a complete fix because it only addresses the value part and not the units. Writing a unit test for this class will demonstrate. It is necessary to change the fundamental behaviour in the `_component` directly because it is there that we have made the assumptions that need to change due to the unforeseen input reported in the defect.

Reply:
ok, sounds good. Will do it.